scripts/check-dependabot-manifests.sh

```
@@ -0,0 +1,36 @@
+# Find all package.json, strip leading ./, ignore node_modules
+all_npm=$(fd --strip-cwd-prefix -t f package.json -E node_modules || true)
+configured=(
+  "package.json"
+  "frontend-pwa/package.json"
+  "packages/tokens/package.json"
+  "packages/types/package.json"
+)
+
+mapfile -t all_npm_arr <<<"$all_npm"
+for p in "${all_npm_arr[@]}"; do
+  if [[ ! " ${configured[*]} " =~ " ${p} " ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)